### PR TITLE
A11y: Provide visual labels or instructions for user input

### PR DESCRIPTION
**Pull request by andrewyu-salesforce**
Schedule Wizard's payment creation table will announce each cell's title label when focused.
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by andrewyu-salesforce** [comment]
Attach To: [W-7965454](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008UEgjIAG/view)

**Comment by andrewyu-salesforce** [comment]
@RadSFDeveloper @npsp-reedestockton The code is ready for review.

File: src/components/UTIL_FormField.component [hunk 1]
```diff
@@ -9,6 +9,8 @@
     <apex:attribute name="showLabel" type="Boolean" default="true" description="When false, hide the field label" />
     <apex:attribute name="overrideRequired" type="Boolean" default="false" description="When true, it will set the field as not required" />
 
+    <apex:attribute name="ariaDescription" type="String" default="" description="override aria label"/>
```
Comment (RadSFDeveloper) [inline src/components/UTIL_FormField.component:12]: The attribute description is not matching the name.

File: src/components/UTIL_FormField.component [hunk 2]
```diff
@@ -78,7 +82,8 @@
                     || $ObjectType[sObjType].fields[field].createable && sObj['Id']=='') && $ObjectType[sObjType].fields[field].type == 'boolean'}">
         <div class="slds-form-element__control">
             <apex:outputLabel styleClass="slds-checkbox" for="inputCB" id="lblCheckbox">
-                <apex:inputField id="inputCB" value="{!sObj[field]}" label="">
+                <apex:inputField id="inputCB" value="{!sObj[field]}" label=""
+                html-aria-description="{!IF(ISBLANK(ariaDescription), '', ariaDescription)}">
```
Comment (RadSFDeveloper) [inline src/components/UTIL_FormField.component:86]: Is `html-aria-description` set by default?

Comment (andrewyu-salesforce) [reply src/components/UTIL_FormField.component]: Update to have the default value label if ariaDescription is missing. aria-description is never set before.

**Comment by andrewyu-salesforce** [comment]
Went through all vf pages with reference of UTIL_FormField and UTIL_InputField and all functions are working as expected.

**Review by RadSFDeveloper — APPROVED** [review]
Looks good, thanks @andrewyu-salesforce !

**Pull request merged**